vendor/modules.txt

```diff
@@ -197,24 +207,25 @@ golang.org/x/net/idna
 golang.org/x/net/internal/timeseries
 golang.org/x/net/trace
 golang.org/x/net/websocket
-# golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+# golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
 golang.org/x/oauth2
 golang.org/x/oauth2/google
 golang.org/x/oauth2/internal
 golang.org/x/oauth2/jws
 golang.org/x/oauth2/jwt
 # golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 golang.org/x/sync/singleflight
-# golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7
+# golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
+golang.org/x/sys/internal/unsafeheader
 golang.org/x/sys/unix
 golang.org/x/sys/windows
-# golang.org/x/text v0.3.2
+# golang.org/x/text v0.3.3
```

Comment:
Note that this bumps golang.org/x/text, thereby fixing CVE-2020-14040 (not sure if we have a BZ for the samples operator on this one).

Reply:
thanks @adambkaplan 

we do have a BZ and I've associated this PR with it, though through the BZ UI vs. marking this PR ... was planning on moving that BZ to MODIFIED manually if need be once this PR merged

could pivot on that and use that BZ here with this PR if we want ... I'm fine either way